buildpipeline/Core-Setup-Publish.json

```diff
@@ -314,10 +429,23 @@
     },
     "TeamName": {
       "value": "DotNetCore"
+    },
+    "VsoToolsRepo": {
```

Comment:
Changed variables to use PB_ correctly.  Putting all the pieces into a package is something we talked about but haven't done yet.  Logged https://github.com/dotnet/buildtools/issues/1598 for refactoring.